cfme/services/workloads.py

```diff
@@ -2,28 +2,79 @@
 """ A model of Workloads page in CFME
 """
 from functools import partial
-from cfme.web_ui import accordion, menu
 
-vm_instances_tree = partial(accordion.tree, "VMs & Instances")
-templates_images_tree = partial(accordion.tree, "Templates & Images")
+from navmazing import NavigateToSibling, NavigateToAttribute  # NOQA
+from cfme.exceptions import DestinationNotFound
+from cfme.web_ui import accordion, match_location, toolbar
+from utils.appliance import Navigatable
+from utils.appliance.implementations.ui import navigator, CFMENavigateStep
 
+vm_instances_tree = partial(accordion.tree, "VMs & Instances", 'All VMs & Instances')
+templates_images_tree = partial(accordion.tree, "Templates & Images", 'All Templates & Images')
 
-@menu.extend_nav
-class services_workloads:
-    def service_vms_instances(_):
-        vm_instances_tree("All VMs & Instances")
+QUADICON_TITLE_LOCATOR = ("//div[@id='quadicon']/../../../tr/td/a[contains(@href,'vm_infra/x_show')"
```

Comment:
yes. I've put it there just in case to not search it - workloads are a bit mysterious. Workloads is a combination of Infra VMs +  Instances / Infra Templates + Images - it might be used for all testing.
But it stays without use from our side at all. 
I'll remove it